packages/plugin/__test__/test.js

```diff
@@ -20,8 +20,8 @@ function processDirectory(dir) {
   const items = readdirSync(dir);
   // Process Files first
   items
-    .filter(item => item.endsWith(".js") || item.endsWith(".ts"))
-    .forEach(filename => {
+    .filter((item) => item.endsWith(".js") || item.endsWith(".ts"))
```

Comment:
why? No problem, just asking. I actually do it without parentheses when there is only one param.

Reply:
prettier did this automatically. Seems that since prettier is active this file hasn't been touched.